### PR TITLE
add some official introduction tutorials

### PR DESCRIPTION
**Pull request by fkromer**
For adding tutorials, packages, or apps
===========================================

##### What did you add to the README.md file?

Have you ensured your contribution adheres to the pull request guidelines of [awesome](https://github.com/sindresorhus/awesome/edit/master/pull_request_template.md)? In particular:

- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Drop all the `A` / `An` prefixes in the descriptions.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.

----------

##### Do you want to be added to the contributors list on the README.md file?  

- [ ] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ A curated list of awesome Dash (plotly) resources
 - [Visualize Earthquakes with Plotly Dash](https://www.giacomodebidda.com/visualize-earthquakes-with-plotly-dash/) - Environmental scan of alternatives to Dash followed with a tutorial.
 - [ARGO Labs - Plotly Dash Tutorial (Video)](https://www.youtube.com/watch?v=yfWJXkySfe0) - Detailed introduction to creating interactive dashboards.
 - [Data Visualization GUIs with Dash and Python (Video playlist)](https://www.youtube.com/watch?v=J_Cy_QjG6NE&list=PLQVvvaa0QuDfsGImWNt1eUEveHOepkjqt) - Five-part series exploring Dash features.
+- [Interactive Image Processing with Dash-Canvas](https://www.youtube.com/watch?v=LKXSFBB5ccI) - Official introduction video to the Dash Canvas components.
+- [Dash Cytoscape Component](https://www.youtube.com/watch?v=snXcIsCMQgk) - Official introduction video to the Dash Cytoscape components.
+- [An introduction to Dash DataTable](https://www.youtube.com/watch?v=dueejcyrYh8) - Official introduction video to the Dash DataTable components.
+- [Introducing the JupyterLab Chart Editor](https://www.youtube.com/watch?v=zR7G2tNVo1Q) - Official introduction video to the JupyterLab Chart Editor.
```
Comment (ucg8j) [inline README.md]: Though I think JupyterLab is fantastic, I don't think it is strictly related to dash. I'd be inclined to remove this from the commit. Thoughts?

Comment (ucg8j) [reply README.md]: Sidenote: Thanks for all the contributions and sorry I didn't see them earlier. Feel free to request me as a reviewer in future so I get a notification to merge them more promptly!

Comment (fkromer) [reply README.md]: You are right. However I experienced JupyterLab as a super helpful development tool for Dash apps. Single parts of the UI can be rapid prototyped with JupterLab in a very efficient manner. Anyway: I can track the JupyterLab video in my notes and remove it from the PR. Just let me know.

Don't worry, PRs are patient 😉. Thx, I'll do so.

Comment (fkromer) [reply README.md]: Quick and dirty fixed.

**Comment by ucg8j** [comment]
Awesome resources :)

**Pull request merged**